h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2004,6 +2004,111 @@ Examples:
+    
+    - In this case, the first bin contains the top 10000 customers with the highest response probability, the second bin contains the next 10000 customers with the highest response probability, and so on.
+
+**Cumulative gains and lift charts** are a graphical representation of the advantage of using a predictive model to choose which customers to target/contact. On the cumulative gains chart, the y-axis shows the percentage of positive responses out of a total possible positive responses. The x-axis shows the percentage of customers contacted. The lift chart shows how much more likely you are to receive responses than if you contacted a random sample of customers.
+
+Example:
+
+.. tabs::
+    .. code-tab:: r R
+
+        # Import the airlines dataset:
+        airlines <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/airlines_train.csv")
+
+        # Build and train the model:
+         model <- h2o.gbm(x = c("Origin", "Distance"), 
+                          y = "IsDepDelayed", 
+                          training_frame = airlines, 
+                          ntrees = 1, 
+                          gainslift_bins = 20)
+
+         # Plot the Gains/Lift chart:
+         gain_table = model@model$training_metrics@metrics$gains_lift_table
+         plot(gain_table$cumulative_data_fraction,
+              gain_table$cumulative_capture_rate,'l')
+
+.. figure:: images/gainslift_plot.png
+    :alt: Gains/Lift Plot
+    :scale: 30%
+
+In addition to the chart, a **Gains/Lift table** is also available. This table reports the following for each group:
+
+- **Cumulative data fractions**: fraction of data used to calculate gain and lift at
+- **Lower threshold**: the lowest score output of the dataset in the data fraction bin
+- **Average response rate**: total number of positive classes / total number of data rows in the dataset
+- **Lift and Cumulative lift**: ratio of cumulative response rate and average response rate.
+- **Score**: the average of all the classifier output probabilities for each individual data fraction bin
+- **Cumulative response rate**: for the first bin, it is the number of positive response over 10000 (assume each bin contains 100000 rows); for the second bin, it is the ratio of the sum of positive response over the first and second bins and 20000; for the third bin, it is the ratio of the sum of positive response over the first, second, and third bins and 30000
+- **Capture rate**: for each bin it is the ratio of positive classes in each bin divided by the total number of positive classes in your dataset
+- **Cumulative capture rate**: for the first bin, it is just the capture rate; the second bin is the ratio of (sum of number of positive classes for the first two bins) and the total number of positive classes in your dataset
+- `Kolmogorov-Smirnov (KS) Metric`_
+- **Gain and Cumulative gain**: calculated for each bin in a cumulative manner: 
```

Comment:
Again I would explain the gain and cumulative gain separately. Also from which source do you take the definition? From code, I see a different definition:

https://github.com/h2oai/h2o-3/blob/a69892d34bf66ec5831be79fd7af93574041a296/h2o-core/src/main/java/hex/GainsLift.java#L185-L186

You can use `lift` instead of `number of positive responses in first bin / number of data in first bin`. The result value is then multiplied by 100.

So my suggestions:

```suggestion
- **Gain**:  :math: 100 \times (lift for current data fraction -1)  
- **Cumulative Gain**:  :math: 100 \times (cumulative lift for current data fraction -1)  
```

Then the table below is not necessary. Or you can use this table at the beginning to explain how cumulative counts work in general. It is the same for cumulative response rate, cumulative score, cumulative capture rate, etc.